aydin/gui/_qt/custom_widgets/constructor_arguments.py

```diff
@@ -30,6 +30,28 @@ def __init__(
         self.arguments_layout.setAlignment(Qt.AlignTop)
 
         for index, (name, default_value) in enumerate(zip(arg_names, arg_defaults)):
+
+            # First we get the description:
+            if inspect.isclass(reference_class):
+                doc = docstring_parser.parse(reference_class.__init__.__doc__)
+            else:
+                doc = docstring_parser.parse(reference_class.__doc__)
+            description = ""
+            for param in doc.params:
+                if param.arg_name == name:
+                    description = param.description
+
+            if description is not None:
+                # Parameters that are marked with (hidden) in their docstrings are 'hidden:
+                if '(hidden)' in description:
```

Comment:
Do you  like to hide the `hidden`  parameters from documentation page  and our saved json too? or those only meant to be hidden  on GUI  @royerloic ?